roles/common/tasks/ubuntu.yml

```diff
@@ -25,12 +25,8 @@
       ignore_errors: true
 
     - name: Wait until SSH becomes ready...
```

Comment:
Looks good, thanks, but could you change the name as well as it not really reflects that new behaviour of the task?